src/pages/Lending/Pool/components/Repay/RepayConfirm.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, CardFooter, CardHeader, Divider, Flex, Heading, Stack } from '@chakra-ui/react'
+import { btcAssetId } from '@shapeshiftoss/caip'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { useCallback } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { usdcAssetId } from 'test/mocks/accounts'
+import { Amount } from 'components/Amount/Amount'
+import { HelperTooltip } from 'components/HelperTooltip/HelperTooltip'
+import { AssetToAsset } from 'components/MultiHopTrade/components/TradeConfirm/AssetToAsset'
+import { WithBackButton } from 'components/MultiHopTrade/components/WithBackButton'
+import { Row } from 'components/Row/Row'
+import { SlideTransition } from 'components/SlideTransition'
+import { RawText, Text } from 'components/Text'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { LoanSummary } from '../LoanSummary'
+import { RepayRoutePaths } from './types'
+
+export const RepayConfirm = () => {
+  const history = useHistory()
+  const translate = useTranslate()
+  const collateralAsset = useAppSelector(state => selectAssetById(state, btcAssetId))
+  const debtAsset = useAppSelector(state => selectAssetById(state, usdcAssetId))
+  const handleBack = useCallback(() => {
+    history.push(RepayRoutePaths.Input)
+  }, [history])
+  return (
+    <SlideTransition>
+      <Flex flexDir='column' width='full'>
+        <CardHeader>
+          <WithBackButton handleBack={handleBack}>
+            <Heading as='h5' textAlign='center'>
+              <Text translation='Confirm' />
+            </Heading>
+          </WithBackButton>
+        </CardHeader>
+        <Stack spacing={0} divider={<Divider />}>
+          <AssetToAsset
+            buyIcon={collateralAsset?.icon ?? ''}
```

Comment:
Probably want to early return in case collateral or debt asset is undefined ? Which it shouldn't but tsc